### ISSUE TITLE
Stats page won't load in DataCite Commons

### ISSUE THREAD

**Issue by MaryHirsch** [datacite/datacite#1441]
#### Expected Behaviour

Stats page visible in DataCite Commons

#### Current Behaviour

Stats page doesn't load

#### Steps to Reproduce

1. Go to https://commons.datacite.org/statistics

### Context (Environment)

DataCite Commons

#### Possible Implementation

Possibly breaking up the query? https://www.apollographql.com/docs/react/v2/performance/performance/

**Comment by MaryHirsch** [comment]
@codycooperross i think this is fixed?

**Comment by codycooperross** [comment]
Yes—I believe this duplicates this https://github.com/datacite/datacite/issues/1897

**Comment by codycooperross** [comment]
Re-opening this issue because the change that fixed it was reverted. The change is described here: https://github.com/datacite/datacite/issues/1897 The change was reverted because it would break the build process of akita or at least make it unstable due to a taxing call to GraphQL that makes multiple calls to Elasticsearch.

**Comment by KellyStathis** [comment]
Closing as this page was removed.
